client/dist/styles.css

```diff
@@ -1,91 +1,157 @@
 }
 
-
 @font-face {
-  font-family: 'Airbnb Cereal App Extra Bold';
+  font-family: 'Cereal';
   font-style: normal;
-  font-weight: normal;
+  font-weight: 800;
   src: local('Airbnb Cereal App Extra Bold'), url('./fonts/airbnb-cereal-app-cufonfonts-woff/AirbnbCerealExtraBold.woff') format('woff');
 }
 
-
 @font-face {
-  font-family: 'Airbnb Cereal App Black';
+  font-family: 'Cereal';
   font-style: normal;
-  font-weight: normal;
+  font-weight: 900;
   src: local('Airbnb Cereal App Black'), url('./fonts/airbnb-cereal-app-cufonfonts-woff/AirbnbCerealBlack.woff') format('woff');
 }
 
-.parent{
+/* layout containers */
+body {
+  margin: 0;
+  padding-top: 25px;
+}
+
+.parent {
+  display: flex;
+  flex-direction: row;
+  border: solid 2px black;
+  height: 442px;
+  width: 100%;
+}
+
+.leftPhoto {
+  display: flex;
+  height: 100%;
+  width: 50%;
+  background-color: #ded8c9;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.rightPhotosContainer{
   display: flex;
   flex-direction: column;
-  align-items: flex-start;
-  background-color: blue;
-  border: solid 3px black;
-  height: 295px;
+  height: 100%;
+  width: 50%;
+}
+
+.rightTopPhotoContainer {
+  display: flex;
+  flex-direction: row;
+  height: 50%;
   width: 100%;
-  background-image: url('../../hero-placeholder.png');
-  background-position: left;
-  background-size: contain;
+}
+
+.rightBottomPhotoContainer {
+  display: flex;
+  flex-direction: row;
+  height: 50%;
+  width: 100%;
+}
+
+.rightTopLeftPhoto {
+  height: 100%;
+  width: 50%;
+  background-color: #80bbc5;
   background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
 }
 
+.rightTopRightPhoto {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 50%;
+  background-color: salmon;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.rightBottomLeftPhoto {
+  height: 100%;
+  width: 50%;
+  background-color: #9fcbd3;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.rightBottomRightPhoto {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 50%;
+  background-color: #c0dcaf;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+/* buttons and button containers */
 .shareAndSaveContainer {
   display: flex;
-  flex-direction: row;
+  flex-direction: row-reverse;
+  margin-top: 35px;
   width: 100%;
-  margin-top: 3%;
-  padding-left: 77%;
 }
 
-.saveButton {
+.saveButtonContainer {
   margin-left: 20px;
+  margin-right: 30px;
 }
 
-.viewPhotosButton {
+.viewPhotosButtonContainer {
   display: flex;
-  flex-direction: row;
+  flex-direction: row-reverse;
   width: 100%;
-  margin-top: 20.5%;
-  padding-left: 85%;
+  margin-top: 160px;
 }
 
-.homeScreenButton {
+.homeScreenButtons {
```

Comment:
You should keep this singular. When we name a class (in any context of programming), notice that we always use the singular form of the word.

Example: Your App component is called `App`, your photo component is called `Photo`, even if you intend to create many instances of that class.